src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -146,12 +146,12 @@ private FormattableString BuildDescription()
 </item>";
                 }
                 FormattableString defaultApiVersion = $"";
-                if (operation.Operation.ApiVersions.Any())
+                if (operation.Operation.Parameters.Where(p => p.IsApiVersion).Any())
                 {
                     defaultApiVersion = $@"
 <item>
 <term>Default Api Version</term>
-<description>{string.Join(", ", operation.Operation.ApiVersions.Select(v => v.Version))}</description>
+<description>{string.Join(", ", operation.Operation.Parameters.Where(p => p.IsApiVersion).Select(x => x.DefaultValue?.Value))}</description>
```

Comment:
This is causing [description change](https://github.com/Azure/azure-sdk-for-net/pull/44148/files#diff-3aa60dfdb9a3c22c0d6e23cb9a6e5b33738d46bb939af869191bb273e7c0c9d8L67-L70), because we don't store api versions in operation anymore, only in operation parameters, some operation doesn't have api version parameter, but have operation-wise api versions.